Benchmarks_on_OVS_integration_approaches.md

```diff
@@ -0,0 +1,213 @@
+# Benchmarks on OVS integration approaches
+
+OpenStack used to use the legacy Nova-network to provide a basic flat network model. However in its successor Neutron, this model is extended as a flat network cannot meet the requirements of a modern data center. Neutron uses a single bridge called the “integration bridge” that all guest instances are connected to provide connectivity to an external network with a physical network interface. This is essentially the same as the nova-network used to connect VMs to a physical interface.
```

Comment:
Please wrap lines.
I use emacs and press alt+q to automatically wrap lines so they are easy to view, comment on and modify.  I'm sure other editors can have a similar function.

"However, in it's successor, Neutron, this model..."
Instead of "Neutron uses a single...", say "XenAPI's integration with Neutron uses a single..."